README.md

```diff
@@ -13,6 +13,13 @@ Basic usage
 
 This package is available as *humanize-duration* on [npm](https://www.npmjs.com/package/humanize-duration) and Bower. You can also include the JavaScript file in the browser.
 
+With `ES Modules`:
+
+```js
+import * as humanizeDuration from 'humanize-duration';
```

Comment:
`humanize-duration` doesn't actually use ES exports...does this kind of thing work on _all_ build setups?